### PR TITLE
FDG-6464 AFG Deficit Explainer - Apply Copy Updates

### PR DESCRIPTION
**Pull request by wyattch**
Coverage: 90.07%
Ticket: https://federal-spending-transparency.atlassian.net/browse/FDG-6464

### CODE REVIEW

**Comment by wyattch** [comment]
PR is currently blocked

File: static/data/glossary/glossary.csv [hunk 1]
```diff
@@ -15,20 +15,20 @@
...
 6,Budget Function,Spending Explainer,"The federal budget is divided into approximately 20 categories, known as budget functions. These categories organize federal spending into topics based on their purpose (e.g., National Defense, Transportation, and Health). These categories can be further broken down into budget sub-functions.",,
-10,Discretionary,Spending Explainer,"Discretionary spending is determined by the president and Congress each year in the budget and appropriations process and includes spending for most defense, education, and transportation programs.",,
-14,Federal Debt,Spending Explainer,The federal debt is the total amount of outstanding borrowing by the U.S. federal government accumulated over the nation’s history. ,,
+10,Discretionary,Spending Explainer,"Discretionary spending is determined by the President and Congress each year in the budget and appropriations process and includes spending for most defense, education, and transportation programs.",,
+14,Federal Debt,Spending Explainer,The federal debt is the total amount of outstanding borrowing by the U.S. federal government accumulated over the nation�s history. ,,
```
Comment (j-hertzler) [inline static/data/glossary/glossary.csv:31]: Needs to be corrected back to an apostrophe

File: static/data/glossary/glossary.csv [hunk 2]
```diff
@@ -15,20 +15,20 @@
...
 31,Surplus,Deficit explainer,When revenue exceeds spending; the difference is a surplus. The federal government uses the surplus primarily to reduce the national debt.,,
 13,Expenditures,Revenue Explainer,"Government expenditure refers to the purchase of goods and services by the government. Such purchases can include public investments and expenses, as well as disbursements and transfer payments consisting of income transfers (i.e. pensions, social benefits) and capital transfer (i.e. construction grants).",,
-33,Trust fund,Revenue Explainer,A type of account for revenue or offsetting revenue dedicated to specific purposes and the spending of that revenue. Some revolving funds are designated as trust funds.,,
+33,Trust fund,Revenue Explainer,A type of account for revenue or offsetting revenue dedicated to specific purposes and the spending of that revenue. Some revolving funds are designated as trust funds�.,,
```
Comment (j-hertzler) [inline static/data/glossary/glossary.csv:25]: Remove strange character here

File: static/data/glossary/glossary.csv [hunk 3]
```diff
@@ -15,20 +15,20 @@
...
-15,Fiscal Year,"Debt, Deficit, Revenue & Spending explainer","The fiscal year is an accounting period spanning 12 months. For the federal government, it runs from October 1 to September 30. For example, Fiscal Year 2022 (FY 2022) started October 1, 2021, and ended September 30, 2022. A fiscal year may be broken down into quarters. For the federal government these quarters are: Q1: October – December, Q2: January – March, Q3: April – June, and Q4: July – September.",,
+15,Fiscal Year,"Debt, Deficit, Revenue & Spending explainer","The fiscal year is an accounting period spanning 12 months. For the federal government, it runs from October 1 to September 30. For example, Fiscal Year 2022 (FY 2022) started October 1, 2021, and ended September 30, 2022. A fiscal year may be broken down into quarters. For the federal government these quarters are: Q1: October � December, Q2: January � March, Q3: April � June, and Q4: July � September.",,
```
Comment (j-hertzler) [inline static/data/glossary/glossary.csv:18]: Correct characters here

File: static/data/glossary/glossary.csv [hunk 4]
```diff
@@ -1,7 +1,7 @@
...
-7,Calendar Year,Debt explainer,This is the one-year period between January 1 and December 31. This is different from the U.S. Government’s fiscal year.,,
+id,term,site_page,definition,url_display,url_path
+4,Bills,"Debt explainer, Deficit Explainer",Bills are short-term government securities with maturities ranging from a few days to 52 weeks. For more information on Treasury bills visit Treasury Direct.,Treasury Direct,https://www.treasurydirect.gov/indiv/products/prod_tbills_glance.htm
+5,Bonds,"Debt explainer, Deficit Explainer","Bonds are government securities that pay interest every six months and mature in 20 or 30 years. For more information on Treasury bonds, including how to purchase them from the Treasury, visit Treasury Direct.",Treasury Direct,https://www.treasurydirect.gov/indiv/products/prod_tbonds_glance.htm
+7,Calendar Year,Debt explainer,This is the one-year period between January 1 and December 31. This is different from the U.S. Government�s fiscal year.,,
```
Comment (j-hertzler) [inline static/data/glossary/glossary.csv:4]: Needs to be corrected back to an apostrophe

**Review by j-hertzler — CHANGES_REQUESTED** [review]
A couple of weird glossary issues to correct but otherwise looks good

**Review by j-hertzler — APPROVED** [review]
Changes look good

**Pull request merged**